### DISCUSSION TITLE
regarding M3U8 playlists

### DISCUSSION THREAD

**Discussion by Vinay-Ban** [4gray/iptvnator#263] [Q&A]
hi. I have Iptvnator 0.13.0.exe installed in my laptop, but I'm unable to play m3u8 file through Iptvnator. neither playlist nor m3u8 text are being played. but same m3u8 file is playing in vlc media player, so I know there is not any issue in my playlist. how to fix it? files which are being played in vlc, are not playing in Iptvnator, and the files which are being played in Iptvnator, are not playing in vlc. both m3u and m3u8.

**Comment by 4gray** [comment]
Please try to use MPV player.

**Reply by redge73** [reply]
Hi, iptvnator 0.14 able to play like this: udp://@239.101.10.46:1234 only with mpv (older iptvnator unable to play it)
iptv nator 0.14 unable to play outside with mpv player like this: rtp://239.186.68.3:10000 
but mpv player is able to play it from command line like this mpv play rtp://239.186.68.3:10000 with overload of artefacts (unwatchable).
Almost 99% no artefacts with vlc except from sender, 
mpv missing something or require another command option for play without artefacts ?
